src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -151,11 +161,38 @@ public AxeView(
         builder.isEnabled(),
         builder.isImportantForAccessibility(),
         builder.labeledBy(),
+        builder.packageName(),
         builder.text(),
         builder.children()
     );
   }
 
+  /**
+   * Recurse through the view hierarchy and grab the package name of the first
+   * non Android System UI view.
+   * @return A non Android System UI packageName.
+   */
+  @SuppressWarnings("WeakerAccess")
+  public String appIdentifier() {
+
+    final StringBuilder result = new StringBuilder();
+
+    forEachRecursive(instance -> {
+
+      result.setLength(0);
+
+      result.append(instance.packageName);
+
+      if (instance.className.endsWith("ContentFrameLayout")) {
```

Comment:
Overlays lack the ContentFrameLayout. This would break our axe scans as well, so I am not worried about it breaking this functionality, but we should have a default case for nothing being found.